Lib/defcon/objects/layer.py

```diff
@@ -251,7 +251,7 @@ def _insertGlyph(self, glyph, beginObservations=True):
     # -------------
 
     def __iter__(self):
```

Comment:
> also, the comment says "Dict Behavior" but real dicts don't behave like that.

Ah, yeah. This is *very old* behavior. It goes back to RoboFog. We can take away the dict comment or note that it's not exactly dict behavior. In these objects, in practice, it's much nicer to iterate over the objects, not the keys.